src/app/App.js

```diff
@@ -17,8 +20,16 @@ export default class AppWrapper {
     this.txParams = txParams
   }
 
+  static async fetch() {
+    return await AppProvider.from.apply(AppProvider, arguments);
```

Comment:
`AppProvider.from(...arguments)` [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) is cleaner here (assuming it's supported by the current babel presets)

Reply:
Really like having fetch and deploy available here btw